base64.gemspec

```diff
@@ -13,9 +13,7 @@ Gem::Specification.new do |spec|
   spec.metadata["homepage_uri"] = spec.homepage
   spec.metadata["source_code_uri"] = spec.homepage
 
-  spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  end
+  spec.files         = ["lib/base64.rb"]
```

Comment:
Can you add `README.md` and `LICENSE.txt` to this?

Reply:
Thanks! Yes, I'll squash the commit after this.